wd_helper.au3

```diff
@@ -1414,12 +1414,17 @@ Func _WD_UpdateDriver($sBrowser, $sInstallDir = Default, $bFlag64 = Default, $bF
 							Else
 								; delete webdriver from disk before unpacking to avoid potential problems
 								FileDelete($sInstallDir & $sDriverEXE)
+								Local $bEXEWasFound = False
 								For $FileItem In $FilesInZip ; Check the files in the archive separately
-									If StringRight($FileItem.Name, 4) = ".exe" Then ; extract only EXE files
+									; https://docs.microsoft.com/pl-pl/windows/win32/shell/folder
+									; https://docs.microsoft.com/pl-pl/windows/win32/shell/folder-items
+									; https://docs.microsoft.com/pl-pl/windows/win32/shell/folderitem
+									If StringRight($FileItem.Name, 4) = ".exe" Or StringRight($FileItem.Path, 4) = ".exe" Then ; extract only EXE files
```

Comment:
I wonder if you could simply check `.Path` instead of `.Name`? I guess it doesn't cause any harm to leave both checks in place.

Reply:
> I guess it doesn't cause any harm to leave both checks in place.

This was also my idea, so I left both options.